app.js

```diff
@@ -1,6 +1,12 @@
 // grab DOM elements
-
+const nameInput = document.getElementById('name-input');
```

Comment:
Should have validated that the HTML element was actually being brought in by logging nameInput here.